optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/PlannerBenchmarkRunner.java

```diff
@@ -338,4 +378,23 @@ public long calculateTimeMillisSpent() {
         return now - startingSystemTimeMillis;
     }
 
+    private static final class ConfigBackup {
```

Comment:
A: +1, short, simple, concise 
B: -1, too long
C: -1, originally, I had something like BackupSolverConfigParts, but I found it to be quite unreadable, just as (C)

D1: OriginalConfigBackup
D2: OriginalBenchmarkConfigBackup